R/formatter.r

```diff
@@ -740,18 +740,15 @@ pvalue <- function(x, accuracy = .001, decimal.mark = ".", add_p = FALSE) {
 
 #' Bytes formatter: convert to byte measurement and display symbol.
 #'
-#' @return a function with three parameters, `x``, a numeric vector that
-#'   returns a character vector, `symbol` the byte symbol (e.g. "Kb")
-#'   desired and the measurement `units` (traditional `binary` or
-#'   `si` for ISI metric units).
+#' @return a function with three parameters, `x``, a numeric vector that returns
```

Comment:
Can you please fix the ``` `` ``` ? (it's not your fault, but you've touched this line so you might as well fix it)